.github/workflows/release.yml

```
@@ -2,97 +2,38 @@ name: Release
 on:
   push:
     tags:
-      - '[5-9].[0-9]+.[0-9]+'
-      - '[5-9].[0-9]+.[0-9]+-*'
+      - '[0-9].[0-9]+.[0-9]+'
+      - '[0-9].[0-9]+.[0-9]+-*'
```

Need the `+` for the major in case we ever hit 2-digit version numbers.
```suggestion
      - '[0-9]+.[0-9]+.[0-9]+'
      - '[0-9]+.[0-9]+.[0-9]+-*'
```